packages/chain/contracts/Eth_broker.sol

```diff
@@ -263,7 +262,7 @@ contract Eth_broker {
         mutex()
         returns (bool)
     {
-        (uint256 daiNeeded, uint256 ethReceived) = _commonMint(
+        (uint256 daiNeeded, uint256 ethReceived, uint256 ethSpent) = _commonMint(
```

Comment:
ethReceived is an unused variable now, better don't assign it
```solidity
(uint256 daiNeeded, , uint256 ethSpent) =
```

Reply:
Not part of this changeset, but I notice that the comments to the internal function `_commonMint` don't document the last return value (`ethSpent`).